kaggle.py

```diff
@@ -3,65 +3,93 @@
 from torch.utils.data import DataLoader
 from tqdm import tqdm
 
-from utils.data import ForecastDataset
+from utils.data import data_frames, ForecastDataset
 from nn import Model
 
 
-def infer(model, loader):
-    """Infer the unit sales with the model.
+def infer(model, loader, window_size):
+    """Infer unit sales of next days with the model.
```

Comment:
possibly add something about the finding of missing sales in the projections, taking the average of the [projected sales] and adding them to the new projections..?

Reply:
So apparently, the rolling average does nothing. You get the exact same projections. Which makes sense, if you think about it for two hours. I'll make the function easier without the rolling average functionality.